src/main/java/pixelperfect/event/Event.java

```diff
@@ -7,17 +7,28 @@
  *
  */
 public class Event {
```

Comment:
Aknowledged, please refer to our UML class diagram to see that I already took some measures in designing the class to make sure it will not end up as a data class.